src/styles/colors.tsx

```diff
@@ -23,6 +23,7 @@ export enum Colors {
   gray1 = '#F8F9F9',
   light = '#FFFFFF',
   purple = '#D39CFE',
+  pink = '#FFC8C8',
```

Comment:
is this going to be part of the color palette officially? if it's for one time use, i think we should just keep it in the component styles